calculator.js

```diff
@@ -1,5 +1,79 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+const prompt = require('prompt');
+
+let userInput = {
```

Comment:
Since you're not reassigning `userInput` but rather just it's properties you can do this:

```suggestion
const userInput = {
```